tests/client/test_custom_code_operators.py

```diff
@@ -94,23 +96,25 @@ def test_post_predict_delete_custom_code_operator_model(channel):
             ),
         ]
 
-        req = service_pb2.PostModelOutputsRequest(model_id=model_id, inputs=inputs)
-        response = post_model_outputs_and_maybe_allow_retries(
-            stub=stub, request=req, metadata=metadata()
+        post_model_outputs_request = service_pb2.PostModelOutputsRequest(
+            model_id=model_id, inputs=inputs
+        )
+        post_model_outputs_resp = post_model_outputs_and_maybe_allow_retries(
+            stub=stub, request=post_model_outputs_request, metadata=metadata()
         )
 
-        raise_on_failure(response)
-        assert len(response.outputs) == 2
-        assert len(response.outputs[0].data.metadata) == 1
-        assert len(response.outputs[1].data.metadata) == 1
+        raise_on_failure(post_model_outputs_resp)
+        assert len(post_model_outputs_resp.outputs) == 2
+        assert len(post_model_outputs_resp.outputs[0].data.metadata) == 1
+        assert len(post_model_outputs_resp.outputs[1].data.metadata) == 1
         assert (
-            response.outputs[0].data.metadata["processed_at"]
-            < response.outputs[1].data.metadata["processed_at"]
+            post_model_outputs_resp.outputs[0].data.metadata["processed_at"]
+            < post_model_outputs_resp.outputs[1].data.metadata["processed_at"]
         )
 
     finally:
-        raise_on_failure(
-            stub.DeleteModel(
-                service_pb2.DeleteModelRequest(model_id=model_id), metadata=metadata()
-            )
+        delete_mv_response = stub.DeleteModelVersion(
```

Comment:
The `try` starts before the `version_id` is assigned, so this may fail unexpectedly. We might want to move the `try` to after the `version_id` is assigned